hudi-flink/src/main/java/org/apache/hudi/operator/StreamWriteOperatorCoordinator.java

```diff
@@ -153,6 +152,14 @@ public void checkpointCoordinator(long checkpointId, CompletableFuture<byte[]> r
   public void checkpointComplete(long checkpointId) {
     // start to commit the instant.
     checkAndCommitWithRetry();
+    // start new instant.
+    startInstant();
+  }
+
+  private void startInstant() {
+    this.inFlightInstant = this.writeClient.startCommit();
```

Comment:
IMO, `instant` or `requestInstant` or `instantTimestamp` sounds better than `inFlightInstant `. wdyt?

Reply:
What do you think about this review suggestion?